Lesson5/src/come/work/home/model/Truck.java

```diff
@@ -0,0 +1,46 @@
+package come.work.home.model;
+
+import java.util.Scanner;
+
+public class Truck extends LandTransport{
+    private int loadCarrier;
+
+    public Truck(String model, int capacity, int body, int speed,int power, int wheels, int fuelConsumption, int loadCarrier) {
+        super(model, capacity, body, speed, power, wheels, fuelConsumption);
+        this.loadCarrier = loadCarrier;
+    }
+    public void infoAutoTruck(){
```

Comment:
i'd suggest to declare method info in Transport, and override it in every realization. In that way you would be able to get an information about all types of transport using one method